dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/InstallServerTest.java

```diff
@@ -155,6 +156,28 @@ public void testInvalidMultiVersionFeatures() throws Exception {
 	checkCommandOutput(po, 21, "CWWKF1405E", null);
 	Log.exiting(c, METHOD_NAME);
     }
+    
+    /**
+     * Test the install of versionless servlet from maven central. Multi-version is not
+     * supported with installServerFeature as it cannot be installed to same
+     * resource.
```

Comment:
```suggestion
     * Test the install of versionless servlet from maven central. "jakartaee-10.0" and "servlet" configuration should install servlet-6.0.
```

Reply:
We'll need to time any special tetss that actually go out to Maven Central - because these haven't been published yet